Makefile

```diff
@@ -53,3 +54,6 @@ install:
 uninstall:
 	/bin/rm -f $(DESTDIR)/$(MODULE_DESTDIR)/nv_peer_mem.ko
 	if [ ! -n "$(DESTDIR)" ]; then $(DEPMOD) -r -ae $(KVER);fi;
+
+gen_nv_symvers:
+	$(PWD)/create_nv.symvers.sh $(KVER)
```

Comment:
KVER  is initialized in the Makefile (for manual build), and also in specfile and debian/rules files for RPM/DEB packaging.